demo/paging/paging-virtual.component.ts

```diff
@@ -40,6 +40,7 @@ export class VirtualPagingComponent {
 
   page = new Page();
   rows = new Array<CorporateEmployee>();
+  //rows = new Array<any>();
```

Comment:
Every change on demo/paging/paging-virtual.component.ts should have not been pushed through. I started using this demo to try out row grouping functionality before I actually created the exclusive demo.